package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "webshark-frontend",
+  "version": "0.1.0",
```

Comment:
version string should be `0.0.1` - Update it upon every milestone.

Reply:
Okay. Will Resolve in next commit